modules/cypress/concepts/CypressCommands.js

```diff
@@ -2,8 +2,14 @@
 
 import { GeneratingConcept } from '@storefront-x/core'
 import path from 'node:path'
+import fs from 'node:fs/promises'
 
 export default class CypressSupport extends GeneratingConcept {
+  async before() {
```

Comment:
This code is duplicated in multiple concepts. Thats a clear indication that something is most likely wrong.  I see you added some functionality regarding to cleanup of old files/directories to the GeneratingConcept. All Cypress concepts extend GeneratingConcept so why is this additional code needed? What is so special about Cypress concepts?